client/tailscale/tailscale.go

```diff
@@ -87,6 +101,14 @@ func bestError(err error, body []byte) error {
 	return err
 }
 
+func errorMessageFromBody(body []byte) string {
+	var j errorJSON
+	if err := json.Unmarshal(body, &j); err == nil && j.Error != "" {
```

Comment:
if err != nil I think we return `return strings.TrimSpace(string(body))` which is maybe ok, but would print a string containing whatever as coming from Tailscale and telling them to do it.

Reply:
Yes, it does. That string is "cert access denied" today. From:

```go
func (h *Handler) serveCert(w http.ResponseWriter, r *http.Request) {
        if !h.PermitWrite {
                http.Error(w, "cert access denied", http.StatusForbidden)
                return
        }
```